public/app/features/integrated-alerting/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { dataQa } from '@percona/platform-core';
+import { Table } from './Table';
+
+const columns = [
+  {
+    Header: 'test col 1',
+    accessor: 'value',
+  },
+];
+
+const data = [
+  {
+    value: 'test value 1',
+  },
+  {
+    value: 'test value 2',
+  },
+];
+
+describe('Table', () => {
+  it('should render the table correctly', async () => {
```

Comment:
```suggestion
  it('should render the table', async () => {
```
Not sure what correctly means :)